README.md

```diff
@@ -16,8 +16,40 @@ For help getting started with Flutter development, view the
 mobile development, and a full API reference.
 
 ## How to run
-
+- run below command
 ```
 flutter pub get
 flutter packages pub run build_runner build --delete-conflicting-outputs
+```
+
+- create launch.json and add add below
```

Comment:
Please create a subsection for Visual Studio like : 

`### Visual Studio`

Because that file is not necessary for people using Android Studio